Dockerfile

```diff
@@ -0,0 +1,38 @@
+# This Docker Image is used to run the CLI Coexistence Tests
+
+FROM --platform=linux/amd64 ubuntu:latest
+
+ENV TANZU_CLI_CEIP_OPT_IN_PROMPT_ANSWER yes
+
+ARG LEGACY_TANZU_CLI_DIR=/app/legacy-tanzu-cli
```

Comment:
Configure new and legacy paths also as env variables same will be accessed in our code